oxide-vpc/tests/common/icmp.rs

```diff
@@ -140,10 +177,22 @@ pub fn gen_icmpv6_echo_req(
         &EtherMeta { dst: eth_dst, src: eth_src, ether_type: EtherType::Ipv6 };
```

Comment:
Github won't let me comment on it, but I just realized the line setting `pay_len` above has been wrong.

```rust
        pay_len: req.buffer_len() as u16,
```

Since you're already here mind fixing it?